.circleci/config.yml

```diff
@@ -30,6 +30,7 @@ executors:
       # DB URL for the jest tests per ormconfig.test.ts
       TEST_DATABASE_URL: "postgres://bloom-ci@localhost:5432/bloom"
       PARTNERS_PORTAL_URL: "http://localhost:3001"
+      SHOW_MANDATED_ACCOUNTS: "TRUE"
```

Comment:
Just a reminder based on our last Slack convo that since we expect this toggle will remain off in core, we can't turn it on in the main suite. We will need a separate suite like we have for listings-approval.